### ISSUE TITLE
Bugs, as observed by me.

### ISSUE THREAD

**Issue by Sutanup** [TheInsightDevelopers/InsightAndroid#50]
**Describe the bug**
1) The problem with uploading of books still exists. Neither I know whether it(the pdf) is being uploaded or not nor if the query if the pdf needs to verified by the developer to appear on the book list. I have tried uploading books thrice, all in vain.

2) The tagging(@) feature doesn't work(my guess being it hasn't been implemented yet). Kindly employ such a safeguard such that duplicate usernames cannot be used and tagging feature sends a notification. 

3) To try to integrate timestamps within the user's chat bubbles

4) Kindly change the background of the Profile section page. Seems cheap and low res. Plus the proportions of texts are all over place.

5) In the chat section, the portion which is being typed on has a transparent background leading to overlap of the written text on the chat history. Kindly add a bubble or make the chat history go up as the typed part increases in size.

6) The messages that are already sent as chats are not selectable, meaning it cant be copid to clipboard.

**To Reproduce**
1)
    a) Open the App
    b) Click on my Profile at the "Top Right" portion of the screen
    c) Click "book upload"
    d) Select any of your desired books after clicking "Select books"
    e) Notice the "absence of anything happening."

2)
    a) Open the App
    b) Click on the chat button at the "Top left" portion of the screen
    c) On the chatbox write "@Govind {any desired message here}" and press send
    d) Notice the absence of realisation within Govind's mind that he isnt notified about your ping.

3)
    a) Open the App
    b) Click on the chat button at the "Top left" portion of the screen.
    c) Type anything in the chatbox and click "send(button)"
    d) Notice the uncertainty of time within all the members with that chat about the rest of the messages they're viewing.

4)
    a) Open the App
    b) Click on my Profile at the "Top Right" portion of the screen
    c) Notice that the Background of the pafe being viewed is blurry and cheaplike.

5)
    a) Open the App
    b) Click on the chat button at the "Top left" portion of the screen
    c) On the chatbox write "{any desired message here}" 
    d) Observe before sending that your text is overlapping the previous chat history.

6)
    a) Open the App
    b) Click on the chat button at the "Top left" portion of the screen
    c) On the chatbox write "{any desired message here}"  and click "send(button)"
    d) Try to select and copy the text of the message you just sent.
    e) Realise you faliure in doing so.

**Expected behavior**
1) Books would be uploaded no problems and a upload progress bar would be visible.

2) I or anyone would be notified only when I oe anyone is tagged respectively.

3)  Timestamps to the chat bubbles would be visible on each of them.

4) Background of the profile section eould look decent or proper.

5) My written messages wouldn't overlap withe the chat history like in popular messaging apps.

6) I would be able to copy what i send or receive via the general. chat.

**Screenshots**
1) Not applicable

2) 
![Screenshot_20210404-102407](https://user-images.githubusercontent.com/81879137/113499023-f61c8d80-952f-11eb-8c37-61d1d6adcc0a.jpg)

3)
![Screenshot_20210404-102510](https://user-images.githubusercontent.com/81879137/113499032-16e4e300-9530-11eb-9750-aaa8745095b9.jpg)

4)
![Screenshot_20210404-102557](https://user-images.githubusercontent.com/81879137/113499037-3845cf00-9530-11eb-8628-eab50c1cde72.jpg)

5)
![Screenshot_20210404-102740](https://user-images.githubusercontent.com/81879137/113499064-73480280-9530-11eb-8586-600921f5e858.jpg)

6) Can't upload pictures of not being able to select a message

**Smartphone (please complete the following information):**
 - Device Name : OnePlus Nord 
 - OS: Oxygen OS 11.1.1.2.AC01DA; Android 11

**Additional context**
Spent too much time writing this.

**Comment by Amar-2003** [comment]
@Sutanup , did remove one issue which was already posted(the third issue)

**Comment by Amar-2003** [comment]
@sksusha8853 , Please take a look at the 4th issue mentioned

**Comment by Sutanup** [comment]
> @Sutanup , did remove one issue which was already posted(the third issue)

@Amar-2003 then I guess the bug fix will be seen in the next update.

**Comment by Amar-2003** [comment]
@Amar-2003 , I have already the issue #34 assigned it me . So lets hope I will be able to do it

**Comment by Amar-2003** [comment]
and @sksusha8853 has done the 4th issue

**Comment by Amar-2003** [comment]
> > @Sutanup , did remove one issue which was already posted(the third issue)
> 
> @Amar-2003 then I guess the bug fix will be seen in the next update.

And I guess the next update won't be be close, since much of the issues weren't solved and We are a bit busy @Sutanup

**Comment by Sutanup** [comment]
yes @Amar-2003 No worries, I'll be waiting. Just ping @Gkm4430 to remind me to update the app.

**Comment by Amar-2003** [comment]
Sure

**Comment by Amar-2003** [comment]
Issue #34 couldn't be done in this update I suppose

**Comment by Amar-2003** [comment]
@Sutanup , we have done quite few suggestions and some imp ones are still , isnn't done like the Chat time stamp and hyperlinking

**Comment by Sutanup** [comment]
Ohh okok. I'll test the update, pls remind @Gkm4430 to ping me personally to update the app.

**Comment by Amar-2003** [comment]
OK
